.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: "ECR-ECS-Deployment"
+on:
+  push:
+    branches:
+      - 792-intern
+  
+jobs:
+  ECR_ECS_Deployment:
+    name: Build Docker Image And Push To ECR 
+    runs-on: ubuntu-latest  
+    steps:  
+        - name: Checkout
+          uses: actions/checkout@v2    
+          
+        - name: Setup AWS ECR Details
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+            aws-region: us-east-1
+
+        - name: Login to Amazon ECR
+          id: login-pf-aws-ecr
+          uses: aws-actions/amazon-ecr-login@v1
+          
+        - name: Build and push the tagged docker image to Amazon ECR
+          id : containerImage
+          env:
+            ECR_REGISTRY: ${{ steps.login-pf-aws-ecr.outputs.registry }}
+            ECR_REPOSITORY: ${{secrets.AWS_ECR_REPO}}
+            IMAGE_TAG: latest
```

Comment:
If the tag is latest, when user pushes new code will the docker image be replaced or new one will be created ?

Reply:
using the latest tag means the most recent build will always be tagged latest in ECR, replacing any previous image with that tag